ec2rlcore/s3upload.py

```diff
@@ -161,7 +136,7 @@ def s3upload(presigned_url, filename):
         # The response puts the URL string in double quotes so cut off the first and last characters
         with open(filename, "rb") as payload:
             response = requests.put(url=presigned_url, data=payload)
-            if response.status_code == 200:
+            if(response.ok):
```

Comment:
The parentheses are redundant (a space works in their place).

Reply:
I'm aware, this is more a personal preference but I have no problem making the changes to meet the projects requirements.